File: proga/1_semestr/labs/lab3/src/main/java/dev/lysmux/lab3/entity/ray/RayKind.java
package dev.lysmux.lab3.entity.ray;

public enum RayKind {
    LIGHT("света"),
    ULTRAVIOLET("ультрафиолетовых"),
    INFRARED("инфракрасных"),
    COSMIC("космических");

    private final String value;

    RayKind(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: proga/1_semestr/labs/lab3/src/main/kotlin/dev/lysmux/lab3/entity/ray/RayKind.kt
package dev.lysmux.lab3.entity.ray

enum class RayKind(private val value: String) {
    LIGHT("света"),
    ULTRAVIOLET("ультрафиолетовых"),
    INFRARED("инфракрасных"),
    COSMIC("космических");

    fun getValue() = value
}